xbmc/addons/Scraper.cpp

```diff
@@ -816,6 +816,20 @@ void DetailsFromFileItem<CArtist>(const CFileItem &item, CArtist &artist)
     artist.discography.emplace_back(discoAlbum);
   }
 
+  int nVideolinks = item.GetProperty("artist.videolinks").asInteger32();
+  artist.videolinks.reserve(nVideolinks);
+  for (int i = 0; i < nVideolinks; ++i)
+  {
+    std::stringstream prefix;
+    prefix << "artist.videolink" << i + 1;
```

Comment:
You could avoid this + 1 operation per for loop iteration if you rewrite the for loop `for (int i = 1; i <= nVideolinks; ++i)` if you follow my previous suggestion to check the number of links before entering the loop.

Reply:
Done.